resources/skins.snapwikiskin.search/App.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <wvui-typeahead-search
+    id="searchform"
+    ref="searchForm"
+    :client="getClient"
+    :domain="domain"
+    :footer-search-text="$i18n('searchsuggest-containing').text()"
+    :suggestions-label="$i18n('searchresults').text()"
+    :accesskey="searchAccessKey"
+    :title="searchTitle"
+    :placeholder="searchPlaceholder"
+    :aria-label="searchPlaceholder"
+    :initial-input-value="searchQuery"
+    :button-label="$i18n('search').text()"
+    :form-action="action"
+    :search-language="language"
+    :show-thumbnail="showThumbnail"
+    :show-description="showDescription"
+  >
+    <input type="hidden" name="title" value="Special:Search" />
+  </wvui-typeahead-search>
+</template>
+
+<script>
+/* global SubmitEvent */
+var wvui = require("wvui");
+module.exports = {
+  name: "App",
```

Comment:
How about a better name?

Reply:
I think "App" might be important, I think it has something to do with importing it.